core/remote_worker/main.cpp

```diff
@@ -271,7 +271,8 @@ namespace fc {
     wroutes->insert({"/remote", sector_storage::serveHttp(local_store)});
 
     api::serve(wrpc, wroutes, *io, "127.0.0.1", config.api_port);
-    api::rpc::saveInfo(config.repo_path, config.api_port, "stub");
+    api::rpc::saveInfo(
+        config.repo_path, config.api_port, Buffer{{1, 2, 3, 4}});  // TODO: Auth
```

Comment:
```suggestion
        config.repo_path, config.api_port, config.miner_api.second);
```
may use miner token like lotus-seal-worker, or call miner.AuthNew